pkg/bpf/bpffs.go

```diff
@@ -99,6 +99,8 @@ func OpenAfterMount(m *Map) error {
 		return err
 	}
 
+	log.WithField(logfields.BPFMapName, m.name).Debug("bpffs is not mounted yet; adding to list of maps to open" +
```

Comment:
I prefer log messages to be on one line so we can grep. I'd rather see this structured as:
``` go
log.WithField(logfields.BPFMapName, m.name).
     Debug("bpffs is not mounted yet; adding to list of maps to open once it is mounted")
```

Reply:
Fixed.